content/_data/changelogs/lts.yml

```diff
@@ -4389,6 +4389,317 @@
+        title: JEP-224
+    authors:
+    - timja
+    message: |-
+      Graduate Overall/SystemRead permission to general availability (GA) status.
+
+  - type: major rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Remove deprecated key exchange and MAC algorithms.
+    references:
+      - pull: 4951
+      - url: "https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md#27"
+        title: SSHD module 2.7 changelog
+
+  - type: rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Allow configuring disabled key exchange and MAC algorithms through system properties.
+    references:
+      - pull: 4951
+      - url: "https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md#27"
+        title: SSHD module 2.7 changelog
+
+  - type: rfe
+    category: rfe
+    pull: 4926
+    issue: 63592
+    authors:
+    - fqueiruga
+    message: |-
+      Improve the layout and clarity of the page displayed when jobs are not yet created.
+
+  - type: rfe
+    category: rfe
+    pull: 4920
+    issue: 60972
+    authors:
+    - romenrg
+    message: |-
+      Name the first breadcrumb "Dashboard" for clarity.
+
+  - type: rfe
+    category: rfe
+    pull: 5016
+    authors:
+    - daniel-beck
+    message: |-
+      Remove TFS Plugin suggestion from Setup Wizard.
+
+  - type: rfe
+    category: rfe
+    pull: 4952
+    authors:
+    - daniel-beck
+    message: |-
+      Disable autocomplete of username on login form.
+
+  - type: rfe
+    category: rfe
+    pull: 4950
+    issue: 63795
+    authors:
+    - aHenryJard
+    message: |-
+      Allow users with the <code>Jenkins.MANAGE</code> permission to restart and safe restart Jenkins.
+
+  - type: rfe
+    category: rfe
+    pull: 4905
+    authors:
+    - tszmytka
+    message: |-
+      Do not render parts of a serialized <code>ConsoleNote</code> in truncated log output.
+
+  - type: rfe
+    category: rfe
+    pull: 4910
+    authors:
+    - daniel-beck
+    references:
+      - pull: 4910
+      - url: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy"
+        title: Cross-Origin-Opener-Policy at developer.mozilla.org
+    message: |-
+      Set <code>Cross-Origin-Opener-Policy</code> to <code>same-origin</code>.
+
+  - type: rfe
+    category: rfe
+    pull: 4911
+    issue: 63429
+    authors:
+    - jglick
+    message: |-
+      Avoid losing work in progress by using a new browser tab to open the plugin link in a configuration screen’s inline help.
+
+  - type: rfe
+    category: rfe
+    pull: 4027
+    issue: 57484
+    authors:
+    - Wadeck
+    message: |-
+      Improve the scripting capacity related to the API Token system.
+      Provide a way to configure a fixed/default API Token for admin during installation phase.
+
+  - type: rfe
+    category: localization
```

Comment:
Move all localization to a location between bugfixes and developer changes (like in changelog generator)?

Reply:
Removed localization changes based on Daniel's recommendation that they are too minor for an LTS changelog